bin/docker/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM java:openjdk-7-jre
+FROM alpine:3.3
```

Comment:
In this case I'd actually prefer to stick with an image from the main `java` repo.  Good news is that they actually have a `java:openjdk-7-jre-alpine` image!

Just tried it out and it looks good.  All I did was swap the docker image tag, removed all those debian ENV settings, and lastly just added `RUN apk add --update bash` so that we have bash to run our launch script.  

Ends up at 180.4 MB :scream: